package.json

```diff
@@ -46,6 +46,7 @@
     "@babel/runtime-corejs2": "^7.4.5",
     "ajv": "^6.7.0",
     "core-js": "^2.5.7",
+    "json-schema-merge-allof": "git+https://github.com/mokkabonna/json-schema-merge-allof.git#e43d0f2227476e069667358b86d0e8c91b50a777",
```

Comment:
Why is this a git dependency?

Reply:
It links to my PR here (https://github.com/mokkabonna/json-schema-merge-allof/pull/5), which makes sure that mergeAllOf can be called with `deep: false` and thus only merges a top-level `allOf`. It's not strictly required, but I guess it's a performance improvement.

Reply:
Since it's not strictly required, I'll just remove it since that's better than depending on a git dependency.